### DISCUSSION TITLE
插件仓库的 Web 页面

### DISCUSSION THREAD

**Discussion by AnzhiZhang** [MCDReforged/PluginCatalogue#312] [Ideas]
表头选择排序方式
需要html？

**Comment by AnzhiZhang** [comment]
有空了直接搭网站

**Comment by zyxkad** [comment]
一个实现: <https://github.com/kmcsr/PluginWebPoint> | <https://mcdr.waerba.com>
文档和README等API稳定之后会写

**Comment by AnzhiZhang** [comment]
https://github.com/AnzhiZhang/MCDReforgedWebsite
https://mcdr.zhanganzhi.com/plugins
已完成

**Comment by zyxkad** [comment]
API 文档写完了, 虽然仍未确定是否继续保持当前结构: <https://github.com/kmcsr/PluginWebPoint/blob/master/docs/api/dev.md>

**Comment by alex3236** [comment]
https://github.com/Fallen-Breath/MCDReforged-Website
https://mcdreforged.com/zh-CN/plugins
